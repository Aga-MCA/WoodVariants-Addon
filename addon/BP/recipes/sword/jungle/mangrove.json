{
  "format_version": "1.12",
  "minecraft:recipe_shaped": {
    "description": {
      "identifier": "jungle_mangrove_sword"
    },
    "tags": [
      "crafting_table",
      "aga"
    ],
    "pattern": [
      "P",
      "P",
      "S"
    ],
    "key": {
      "S": {
        "item": "aga_wood_variants:mangrove_stick"
      },
      "P": {
        "item": "aga_wood_variants:jungle_panels"
      }
    },
    "result": {
      "item": "aga_wood_variants:jungle_mangrove_sword"
    }
  }
}