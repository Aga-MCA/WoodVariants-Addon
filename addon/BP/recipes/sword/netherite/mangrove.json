{
  "format_version": "1.17",
  "minecraft:recipe_smithing_transform": {
    "description": {
      "identifier": "netherite_mangrove_sword"
    },
    "tags": [
      "smithing_table"
    ],
    "template": "minecraft:netherite_upgrade_smithing_template",
    "base": "aga_wood_variants:diamond_mangrove_sword",
    "addition": "minecraft:netherite_ingot",
    "result": "aga_wood_variants:netherite_mangrove_sword"
  }
}