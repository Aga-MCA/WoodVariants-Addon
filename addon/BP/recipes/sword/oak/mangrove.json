{
  "format_version": "1.12",
  "minecraft:recipe_shaped": {
    "description": {
      "identifier": "oak_mangrove_sword"
    },
    "tags": [
      "crafting_table",
      "aga"
    ],
    "pattern": [
      "P",
      "P",
      "S"
    ],
    "key": {
      "S": {
        "item": "aga_wood_variants:mangrove_stick"
      },
      "P": {
        "item": "aga_wood_variants:oak_panels"
      }
    },
    "result": {
      "item": "aga_wood_variants:oak_mangrove_sword"
    }
  }
}