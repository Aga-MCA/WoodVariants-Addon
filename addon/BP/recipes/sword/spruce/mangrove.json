{
  "format_version": "1.12",
  "minecraft:recipe_shaped": {
    "description": {
      "identifier": "spruce_mangrove_sword"
    },
    "tags": [
      "crafting_table",
      "aga"
    ],
    "pattern": [
      "P",
      "P",
      "S"
    ],
    "key": {
      "S": {
        "item": "aga_wood_variants:mangrove_stick"
      },
      "P": {
        "item": "aga_wood_variants:spruce_panels"
      }
    },
    "result": {
      "item": "aga_wood_variants:spruce_mangrove_sword"
    }
  }
}