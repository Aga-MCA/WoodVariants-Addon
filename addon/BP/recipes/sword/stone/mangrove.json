{
  "format_version": "1.12",
  "minecraft:recipe_shaped": {
    "description": {
      "identifier": "stone_mangrove_sword"
    },
    "tags": [
      "crafting_table",
      "aga"
    ],
    "pattern": [
      "P",
      "P",
      "S"
    ],
    "key": {
      "S": {
        "item": "aga_wood_variants:mangrove_stick"
      },
      "P": {
        "item": "minecraft:cobblestone"
      }
    },
    "result": {
      "item": "aga_wood_variants:stone_mangrove_sword"
    }
  }
}