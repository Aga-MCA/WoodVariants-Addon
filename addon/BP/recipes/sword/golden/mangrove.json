{
  "format_version": "1.12",
  "minecraft:recipe_shaped": {
    "description": {
      "identifier": "golden_mangrove_sword"
    },
    "tags": [
      "crafting_table",
      "aga"
    ],
    "pattern": [
      "P",
      "P",
      "S"
    ],
    "key": {
      "S": {
        "item": "aga_wood_variants:mangrove_stick"
      },
      "P": {
        "item": "minecraft:gold_ingot"
      }
    },
    "result": {
      "item": "aga_wood_variants:golden_mangrove_sword"
    }
  }
}